CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
+* Kubernetes 1.25 was deprecated on January 14, 2024 and support transitions to [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy). Please upgrade to Kubernetes version 1.26 or above.
+* Starting with Kubernetes 1.29, the default cgroups implementation on Azure Linux AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance, and read the [FAQ](https://learn.microsoft.com/troubleshoot/azure/azure-kubernetes/aks-increased-memory-usage-cgroup-v2) for cgroupsv2.
+* All current AKS API versions silently ignore unknown fields. An unknown field is a field that isn't part of the AKS API. AKS API version 2024-01-01, 2024-01-02-preview and all subsequent API versions will change this behavior. Unknown fields in a request will result in the request being rejected with an error stating that the unknown field is not understood. This change only impacts new API versions and won't impact you unless you update to use an API version 2024-01-01 or later. Existing API calls (via Azure Resource Manager templates or otherwise) will continue to function as-is.
+ 
+### Release notes
+
+* Features
+  * [Planned Maintenance](https://learn.microsoft.com/azure/aks/planned-maintenance) and [node-image Upgrade channel](https://learn.microsoft.com/azure/aks/auto-upgrade-node-os-image) is available in Azure Portal. 
+  * [Associate capacity reservation groups to node pools](https://learn.microsoft.com/azure/aks/manage-node-pools#associate-capacity-reservation-groups-to-node-pools-preview) is now generally available. 
+
+* Preview features
+  * AKS 1.29 is in preview. 
+  * [Control Plane Metrics (API server, ETCD, Cluster Autoscaler, etc)](https://learn.microsoft.com/azure/aks/monitor-control-plane-metrics) are now available in Managed Prometheus for Public preview. 
+ 
+
+* Bug Fixes 
+  * Fix the issue that the new CRD "azureapplicationgatewayrewrites.appgw.ingress.azure.io " is not delivered into the overlay cluster. This problem occurs during cluster upgrades.
```

Comment:
need to remove 'overlay'. issue is unclear on what happens during cluster upgrades with respect to the CRD being mentioned